### PR TITLE
Fix character encoding problem on welcome page.

### LINKED ISSUE #152: The application name is garbled when it contains multibyte characters.
I am going to use Openfire Meetings in Japanese.
I am currently using Openfire 4.6.0 and openfire-pade-plugin 1.2.0 on Windows Server 2016.
When I set Japanese characters for Application Name, Default Remote Display Name, Welcome Page Title, etc. in the Openfire Admin Console,  the Japanese characters are displayed as ???? on Openfire Meetings welcome page.

I think you need to do HTML escaping (e.g. Apache Commons StringEscapeUtils.escapeHtml4) before using JiveGlobals.setProperty or convert character encoding after using JiveGlobals.getProperty.

### PR DESCRIPTION
**Pull request by cool0707**
Fix #152 issue.

### CODE REVIEW

**Comment by gjaekel** [comment]
Wow! Two lines that matters. Thank you for your investigations!

**Comment by deleolajide** [comment]
@cool0707 Thank you for your contribution 👍

**Comment by gjaekel** [comment]
Fixes #155

File: web/src/main/java/org/jivesoftware/openfire/plugin/ofmeet/InterfaceConfigServlet.java [hunk 1]
```diff
@@ -45,6 +45,9 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
         {
             Log.trace( "[{}] interface_config requested.", request.getRemoteAddr() );
 
+            response.setCharacterEncoding( "UTF-8" ); 
+            response.setContentType( "UTF-8" );
```
Comment (gjaekel) [inline web/src/main/java/org/jivesoftware/openfire/plugin/ofmeet/InterfaceConfigServlet.java:49]: Should this be `response.setContentType( "text/html );` ?!?

Comment (deleolajide) [reply web/src/main/java/org/jivesoftware/openfire/plugin/ofmeet/InterfaceConfigServlet.java:49]: It should be "application/javascript"

Comment (gjaekel) [reply web/src/main/java/org/jivesoftware/openfire/plugin/ofmeet/InterfaceConfigServlet.java:49]: Sounds even better ;)

**Pull request merged**